src/LogIn.js

```diff
@@ -1,71 +1,109 @@
 import React, { useState } from "react";
 import axios from "axios";
+import bgImage from './image/background.png';
+import logo from './image/antcolony-logo.png';
+import './index.css';
+import './App.css';
```

Comment:
Not sure what is code convection but everything should start with small or large Later not mixed, so: 
Index.css, App.css or index.css, app.css